OsmAnd/res/values/strings.xml

```diff
@@ -34,14 +34,14 @@
     <string name="monitoring_min_speed">Minimum speed</string>
     <string name="monitoring_min_accuracy">Minimum accuracy</string>
     <string name="monitoring_min_distance">Minimum displacement</string>
-    <string name="tracks_view_path">Menu — My places — Tracks</string>
-    <string name="multimedia_notes_view_path">Menu — My places — Notes</string>
-    <string name="osm_edits_view_path">Menu — My places — OSM Edits</string>
-    <string name="reset_plugin_to_default">Reset plugin settings to default</string>
+    <string name="tracks_view_path">Menu → My places → Tracks</string>
+    <string name="multimedia_notes_view_path">Menu → My places → Notes</string>
+    <string name="osm_edits_view_path">Menu → My places → OSM edits</string>
+    <string name="reset_plugin_to_default">Reset plugin settings to defaults</string>
     <string name="multimedia_rec_split_title">Recorder split</string>
     <string name="multimedia_use_system_camera">Use system app</string>
     <string name="multimedia_photo_play_sound">Camera shutter sound</string>
-    <string name="osm_authorization_success">Authorization is successful</string>
+    <string name="osm_authorization_success">Authorized</string>
```

Comment:
yes, fine.

Reply:
It is however a bit cold.
Can the already present
<string name="logged_as">Logged in as %1$s</string>
be used?

Reply:
'Cold' is well stated. ;) Would need to research where this is used, i.e. if logged in is correct, and if we have a variable there to fill % $1s?